charts/otel-cloud-stack/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+name: otel-cloud-stack
+version: 0.1.0
+description: Helm chart for an Opentelemetry Kubernetes Quickstart
+type: application
+home: https://opentelemetry.io/
+sources:
+  - https://github.com/open-telemetry/opentelemetry-operator
+maintainers:
+  - name: jaronoff97
+  - name: anammedina21
```

Comment:
Please list @TylerHelmuth @dmitryax and @Allex1.  I'd love if you wanted to help maintain this chart and become an approver after it is merged, we did something similar with @puckpuck and the demo chart.

Reply:
can do!